### ISSUE TITLE
What is the point of the 'custom' menu?

### ISSUE THREAD

**Issue by Null-Senpai** [konkor/cpufreq#71]
What is the custom menu for? At first I thought it's some sort of profile setting (for maybe having custom governor and turbo state configurations etc.), but this does not seem to be the case? Unless something is broken on my end.

-Set Governor to Performance
-Set Turbo on
-Set max frequency to the max
-Click "New...", call it "power" and save

-Set Governor to Schedutil
-Set Turbo off
-Set max frequency to ~50%
-Click "New...", call it "everyday" and save

Now, clicking either "power" or "everyday" does nothing at all (except close the menu). Is this supposed to affect the cpu frequency settings? 

PS: Ignoring the custom menu and setting everything manually works just fine.

**Comment by konkor** [comment]
Hi, some information about your system like CPU vendor, cpufreq driver, kernel version, distro. It could help to test it.
Yes, Custom menu is Profiles submenu for quick switching between settings. It's just showing the currently active profile.

**Comment by Null-Senpai** [comment]
I have an Intel i5 3340M, and afaik I am using the acpi driver (using the intel_pstates=disabled kernel parameter). Kernel is Linux 4.14.49-1-lts, running on Antergos (Arch based distribution).
`/proc/cpuinfo` is here: https://hastebin.com/umejozodiv.sql

**Comment by konkor** [comment]
Okay. Can you find and paste here the profile part from the log when you are switching custom profiles?
```
sudo journalctl -f

## It should be something like this:
Jun 21 18:26:37 debian gnome-session[1193]: Loading profile... {"name":"Daily","minf":0,"maxf":100,"turbo":false,"cpu":8,"acpi":true,"core":[{"g":"ondemand","a":1600000,"b":2668000},{"g":"ondemand","a":1600000,"b":2668000},{"g":"ondemand","a":1600000,"b":2668000},{"g":"ondemand","a":1600000,"b":2668000},{"g":"ondemand","a":1600000,"b":2668000},{"g":"ondemand","a":1600000,"b":2668000},{"g":"ondemand","a":1600000,"b":2668000},{"g":"ondemand","a":1600000,"b":2668000}],"guid":"7db2d3a8a25e895a17d2cda45a25d9b9"}
```
Maybe you have something with profiles?...

**Comment by Null-Senpai** [comment]
Okay, I was in the middle of typing this, with logs etc., and then I realized something: It DOES apply the settings, it just takes a while for the gnome-shell extension to show the changes. 

I clicked on the profile, and then clicked the extension right away to see if it applied, but it seems I need to give it some time to refresh. Which I did do, but the content of the drop down menu does not refresh if it's open. 

So basically: The icon seems to refresh.
The text in the drop down that says "Loading 152%" also seems to refresh.

The Turbo toggle, governor setting and Min/Max sliders do NOT refresh.
